packages/odyssey-react-mui/src/labs/DatePickers/DateTimePicker.tsx

```diff
@@ -38,6 +38,11 @@ import {
   DesignTokens,
 } from "../../OdysseyDesignTokensContext.js";
 import { OdysseyThemeProvider } from "../../OdysseyThemeProvider.js";
+import { pxToRem } from "../../theme/index.js";
+
+export const DateTimePickerSize = {
+  maxWidth: `${pxToRem(485.75)}rem`,
+};
```

Comment:
`486` is fine. It rounds up anyway when at 100% and will be rounding regardless even at other percentages, but the rounding is inconsistent depending on the zoom level. `486` will leave it more consistent.